dev-workflow-ui/webContent/includes/template/template.xhtml

```diff
@@ -44,6 +44,12 @@
               </p:commandLink>
             </li>
 
+            <li class="topbar-item">
+              <a href="/system/engine-cockpit" title="Engine Cockpit">
```

Comment:
You should consider base context name ...

Reply:
fuck